gf-repos-2024-09-23.json

```
@@ -3574,5 +3574,13 @@
     "config_files": [
       "config.yaml"
     ]
+  },
+  {
+    "repo_url": "https://github.com/googlefonts/googlesans-code",
+    "rev": "4de3a9eddffd7df57abc0f6f28c83a428329f437",
+    "auth": true,
+    "config_files": [
+      "config.yaml"
```

So we have a bit of a problem here, which is that we _really_ expect config files to be in either a `sources` or `Sources` directory, but this repo uses a `source` directory.

We had a similar issue with GS, where everything was in a `source` directory, and there was no `config.yaml`;  we solved that issue by adding `sources/config.yaml` and then pointing from there to the actual sources located in the `source` directory.

However this repo _has_ a config.yaml, it's just in the wrong place.


Some possible solutions:

- add support in `google-fonts-sources` for sources living in a `source` directory. This wouldn't be too annoying, but it feels a little bad to do this work for a single case, when we really expect fonts to use `source` or `sources`.
- fix this in the GSC repo (rename `source` to `sources`). This seems easy, but I'm not sure what breakage could result.
- fix this in the GSC repo by adding a `sources` directory containing a symlink'd `config.yaml`. This one scares me a bit, because... symlinks. Does anyone use windows?

---

Can we add it and let it fail until one of the solutions has occurred? - I'll follow up with GSC about renaming source

---

we can, although if the fix requires an update in GSC we'll have to PR this again with a rev that includes the fix.

---

https://github.com/googlefonts/googlesans-code/commit/6ba3268574c263baea8c6f665f43a04301f4a3c5 renamed sources so I think we're good here